ecommerce-website/src/SystemVariables.js

```diff
@@ -1,3 +1,3 @@
-const baseUrl = "https://heagle-backend.herokuapp.com";
-// const baseUrl = "http://localhost:3001"
+// const baseUrl = "https://heagle-backend.herokuapp.com";
+const baseUrl = "http://localhost:3001"
```

Comment:
Make sure to comment line 2 and uncomment line 1

Reply:
Yes, you're right! Updated!